mta-v7.x/mta.sw.yaml

```diff
@@ -431,6 +445,20 @@ states:
               topic: "MTA Analysis Workflow"
               link: '$SECRET.mta.url + "/hub/applications/" + (.application.id|tostring) + "/analysis/report"'
               severity: "high"
+    transition: reportCompletionWithMandatoriesIncidentsInApplication
+  - name: reportCompletionWithMandatoriesIncidentsInApplication
+    type: inject
+    data:
+      result:
+        completedWith: error
+        message: '"MTA analysis for " + .application.repository.url + " completed with " + (.application.mandatoriesIncidents|tostring) +" incidents in mandatory issues. Check the analysis report." + (if .migrationWaveJiraTicket != null then " Application was exported to issue manager." else "" end)'
+        output:
+          - key: mtaAnalysisResultURL
+            value: ($SECRET.mta.url + "/hub/applications/" + (.application.id|tostring) + "/analysis/report")
+            format: link
+          - key: migrationWaveJiraTicketURL
```

Comment:
@gabriel-farache Pls replace the key with a human-readable value since it is rendered in the UI, e.g. `Migration Wave Jira ticket URL`
(same goes for the other key)

Reply:
Somehow I missed the outputs key should be human readable: https://github.com/parodos-dev/serverless-workflows/pull/442